docs/reference/migration/migrate_7_7.asciidoc

```diff
@@ -32,6 +27,21 @@ log noise. From 7.7 onwards the default log level for logger in this hierarchy
 is now `INFO`, in line with most other loggers. If needed, you can recover the
 pre-7.7 default behaviour by adjusting your <<logging>>.
```

Comment:
Because this is now being reused in the Installation and Update Guide, any internal xrefs
need to be made into external links. This should fix the docs CI test.
```suggestion
pre-7.7 default behaviour by adjusting your {ref}/logging.html[logging].
```

Reply:
Thank you!!
I did spot the warning, but since it built (and linked) fine locally, I dismissed it as potential failure source (and was just trying to build `--all` to hopefully get more details).